frontend/src/components/PreviewReview/PreviewReviewModal.tsx

```diff
@@ -44,7 +44,14 @@ const PreviewReviewModal = ({
     }
     return "";
   };
-
+  // modalState returns "Published" if published_at has a value and "Draft" if it is null
+  const modalState = () => {
+    if (review.publishedAt == null) {
```

Comment:
I think you also need to check for `undefined` here. Both `undefined` and `null` evaluate to `false`, so I think you can just do something like `if(!review.publishedAt) {...`

Reply:
Thank you! 
I think the `==` uses the juggling-check method so it checks for `null` and `undefined` at the same time, if I say `==null`. (https://www.delftstack.com/howto/typescript/check-for-both-null-and-undefined-in-typescript/#use-the-and-operators-to-check-for-both-null-and-undefined-in-typescript). 

Let me know if it is wrong though!

Reply:
Looks like you're right! :smile: Taught me something lol